2025-01-19 05:54:12 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 05:54:12 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c817372660527028907e0961","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 05:54:12 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 05:54:14 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 05:54:15 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c817372660549741168e0961","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 05:54:15 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 05:54:18 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 05:54:18 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c817372660582281568e0961","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 05:54:18 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 05:54:22 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 05:54:22 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 05:57:56 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c817372662765344273e0961","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 05:57:56 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 05:57:57 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 05:57:57 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c817372662777874400e0961","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 05:57:57 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 05:57:59 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 05:58:00 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c817372662800494654e0961","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 05:58:00 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 05:58:03 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 05:58:03 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701017372662838114578e8e5d","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 05:58:03 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 05:58:07 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 05:58:07 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 05:58:24 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/11/08/13/58/53/124099015_p0.png
2025-01-19 06:02:51 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801317372665717412550e472b","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:02:51 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:02:52 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 06:02:53 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801317372665730222714e472b","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:02:53 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:02:55 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 06:02:55 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801317372665752962952e472b","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:02:55 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:02:58 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 06:02:58 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801317372665785683190e472b","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:02:58 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:03:02 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 06:03:02 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 06:09:33 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /ae6RMYWC%2F889478%2F677f6ef69fac2e6671154da1a09305e557c40e49%2F678c96ff7144ef95129741f9ad8d64f221b6ee52?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5eHA97CnrZRhbeYSVGGlEojeLoV2ZfD0zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUjpMV73q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQzFAznsr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSigdhiM%2FXtuMkagAFz03W2W0QoSojL%2FQpfM%2Fmt0DiRa%2FfqDW%2FDkhwWuB5HzES7CM0Jt027qDwzeHMNmYwIwsQhLlayrL4T54mTEuJ%2B4pu86ZxKXIn9qWSKp0rl9%2FSspz0qYOBNa6ADiRPoP5n6ppidXE%2FTG4Mk5woMlTtqUo03b5%2BsYKVg%2F3Fd7PJjOiAA&uploadId=84E87944B4554EF3A8A2C3F4279DD769&x-oss-access-key-id=STS.NT2Hdvsip2usKw7pJvt596Rh1&x-oss-expires=1737270543&x-oss-signature=UZydY346DHxKNmQBv3AFKqOYl8llcA6P14J7vA5WXzs%3D&x-oss-signature-version=OSS2
2025-01-19 06:11:22 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801417372670826247056e1598","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:11:22 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:11:23 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 06:11:24 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801417372670839007136e1598","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:11:24 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:11:26 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 06:11:26 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801417372670861697311e1598","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:11:26 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:11:29 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 06:11:29 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801417372670894437567e1598","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:11:29 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:11:33 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 06:11:33 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 06:14:30 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f117372672701975866ed3fa","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:14:30 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:14:31 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 06:14:31 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f117372672714446028ed3fa","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:14:31 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:14:33 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 06:14:33 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f117372672736946309ed3fa","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:14:33 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:14:36 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 06:14:37 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f117372672769446742ed3fa","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:14:37 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:14:41 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 06:14:41 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 06:22:33 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /np6z4nLf%2F889478%2F635910d7c251223fc5f0464aa11bdef70e50570c%2F678c9a0a50e5c71d7f944d2d89835e276a943440?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5eHA97CnrZRhbeYSVGGlEojeLoV2ZfD0zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUjpMV73q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQzFAznsr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSigdhiM%2FXtuMkagAFz03W2W0QoSojL%2FQpfM%2Fmt0DiRa%2FfqDW%2FDkhwWuB5HzES7CM0Jt027qDwzeHMNmYwIwsQhLlayrL4T54mTEuJ%2B4pu86ZxKXIn9qWSKp0rl9%2FSspz0qYOBNa6ADiRPoP5n6ppidXE%2FTG4Mk5woMlTtqUo03b5%2BsYKVg%2F3Fd7PJjOiAA&uploadId=A6AA444658824149B140430C0DE34A89&x-oss-access-key-id=STS.NT2Hdvsip2usKw7pJvt596Rh1&x-oss-expires=1737271322&x-oss-signature=g647Gru9qKhgnWxb6h9k578F1E4crWfp89NqMmppZlA%3D&x-oss-signature-version=OSS2
2025-01-19 06:26:56 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c917372680163432618ed791","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:26:56 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:26:57 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 06:26:57 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c917372680176242714ed791","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:26:57 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:26:59 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 06:27:00 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c917372680198992884ed791","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:27:00 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:27:03 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 06:27:03 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c917372680231753114ed791","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:27:03 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:27:07 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 06:27:07 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 06:40:54 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /xyzXcDDc%2F889478%2F642d687d67afdaf0e1074508ad4edbd5236982e6%2F678c9e58d3c4a80638664be88f206c80ba5fb143?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5eHA97CnrZRhbeYSVGGlEojeLoV2ZfD0zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUjpMV73q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQzFAznsr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSigdhiM%2FXtuMkagAFz03W2W0QoSojL%2FQpfM%2Fmt0DiRa%2FfqDW%2FDkhwWuB5HzES7CM0Jt027qDwzeHMNmYwIwsQhLlayrL4T54mTEuJ%2B4pu86ZxKXIn9qWSKp0rl9%2FSspz0qYOBNa6ADiRPoP5n6ppidXE%2FTG4Mk5woMlTtqUo03b5%2BsYKVg%2F3Fd7PJjOiAA&uploadId=975F7F7451BE4864AB0812436CC73C50&x-oss-access-key-id=STS.NT2Hdvsip2usKw7pJvt596Rh1&x-oss-expires=1737272424&x-oss-signature=oZynmrvXMnopR%2BRD2wF15KbjIi34yxwM9P%2BpyWqL1yc%3D&x-oss-signature-version=OSS2
2025-01-19 06:42:42 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06717372689623572184ec746","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:42:42 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:42:43 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 06:42:43 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06717372689636102359ec746","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:42:43 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:42:45 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 06:42:46 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06717372689658772650ec746","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:42:46 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:42:49 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 06:42:49 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06717372689696486438ec744","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:42:49 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:42:53 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 06:42:53 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 06:48:50 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06717372693301182234ec73d","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:48:50 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:48:51 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 06:48:51 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06717372693313712438ec73d","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:48:51 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:48:53 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 06:48:53 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06717372693336452692ec73d","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:48:53 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:48:56 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 06:48:57 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06717372693369003076ec73d","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:48:57 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:49:01 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 06:49:01 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 06:57:10 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /GDrbFtbf%2F889478%2F6777737b6989b1db6d1e4079a667f9a3385851d1%2F678ca22786034c46b5b04a09aa70505511546329?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5eHA97CnrZRhbeYSVGGlEojeLoV2ZfD0zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUjpMV73q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQzFAznsr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSigdhiM%2FXtuMkagAFz03W2W0QoSojL%2FQpfM%2Fmt0DiRa%2FfqDW%2FDkhwWuB5HzES7CM0Jt027qDwzeHMNmYwIwsQhLlayrL4T54mTEuJ%2B4pu86ZxKXIn9qWSKp0rl9%2FSspz0qYOBNa6ADiRPoP5n6ppidXE%2FTG4Mk5woMlTtqUo03b5%2BsYKVg%2F3Fd7PJjOiAA&uploadId=42ECF260D43549A3A583C1F44A606B53&x-oss-access-key-id=STS.NT2Hdvsip2usKw7pJvt596Rh1&x-oss-expires=1737273399&x-oss-signature=z1Er2LfnFOI4hzK5kEF8mZokhuSoPqmWHyEyyU9D%2FZ4%3D&x-oss-signature-version=OSS2
2025-01-19 06:58:12 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f017372698919555484e3c7a","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:58:12 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:58:13 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 06:58:13 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f017372698932305632e3c7a","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:58:13 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:58:15 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 06:58:15 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f017372698955035924e3c7a","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:58:15 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:58:18 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 06:58:18 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f017372698987916230e3c7a","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:58:18 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:58:22 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 06:58:22 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 06:59:48 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c917372699884365876ed789","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:59:48 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:59:49 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 06:59:49 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c917372699896996035ed789","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:59:49 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:59:51 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 06:59:52 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c917372699919556285ed789","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:59:52 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:59:55 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 06:59:55 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c917372699952116574ed789","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 06:59:55 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 06:59:59 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 06:59:59 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 07:04:35 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /I1erDHg8%2F889478%2F6563e4e404b3dac931c04cdabfa35035d88f87ce%2F678ca3e5cb9a8f4d72ea4c5fbb25f12a85f7bcbb?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5eHA97CnrZRhbeYSVGGlEojeLoV2ZfD0zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUjpMV73q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQzFAznsr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSigdhiM%2FXtuMkagAFz03W2W0QoSojL%2FQpfM%2Fmt0DiRa%2FfqDW%2FDkhwWuB5HzES7CM0Jt027qDwzeHMNmYwIwsQhLlayrL4T54mTEuJ%2B4pu86ZxKXIn9qWSKp0rl9%2FSspz0qYOBNa6ADiRPoP5n6ppidXE%2FTG4Mk5woMlTtqUo03b5%2BsYKVg%2F3Fd7PJjOiAA&uploadId=17912C2D7E244FCD9F5CAEE1383ABA6A&x-oss-access-key-id=STS.NT2Hdvsip2usKw7pJvt596Rh1&x-oss-expires=1737273845&x-oss-signature=px%2BaHCJpHFaInMI3bLfYJYV1IEHm4JJQra2HoBoAFdY%3D&x-oss-signature-version=OSS2
2025-01-19 07:10:51 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Read timed out. (read timeout=30)")': /dSAoX6sq%2F889478%2F6735d73332841f81370642618cda50a015a68fc1%2F678ca547e9ac85879d704c479d576a3f2b452058?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5eHA97CnrZRhbeYSVGGlEojeLoV2ZfD0zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUjpMV73q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQzFAznsr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSigdhiM%2FXtuMkagAFz03W2W0QoSojL%2FQpfM%2Fmt0DiRa%2FfqDW%2FDkhwWuB5HzES7CM0Jt027qDwzeHMNmYwIwsQhLlayrL4T54mTEuJ%2B4pu86ZxKXIn9qWSKp0rl9%2FSspz0qYOBNa6ADiRPoP5n6ppidXE%2FTG4Mk5woMlTtqUo03b5%2BsYKVg%2F3Fd7PJjOiAA&uploadId=763A009C1EFE402A85BD4EC33CA70ABA&x-oss-access-key-id=STS.NT2Hdvsip2usKw7pJvt596Rh1&x-oss-expires=1737274199&x-oss-signature=2t2FAhdY0RCDobUyyGWWnj1Zn%2B2R5Fv5WNkkpWk9N%2FU%3D&x-oss-signature-version=OSS2
2025-01-19 07:10:54 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/dSAoX6sq%2F889478%2F6735d73332841f81370642618cda50a015a68fc1%2F678ca547e9ac85879d704c479d576a3f2b452058?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5eHA97CnrZRhbeYSVGGlEojeLoV2ZfD0zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUjpMV73q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQzFAznsr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSigdhiM%2FXtuMkagAFz03W2W0QoSojL%2FQpfM%2Fmt0DiRa%2FfqDW%2FDkhwWuB5HzES7CM0Jt027qDwzeHMNmYwIwsQhLlayrL4T54mTEuJ%2B4pu86ZxKXIn9qWSKp0rl9%2FSspz0qYOBNa6ADiRPoP5n6ppidXE%2FTG4Mk5woMlTtqUo03b5%2BsYKVg%2F3Fd7PJjOiAA&uploadId=763A009C1EFE402A85BD4EC33CA70ABA&x-oss-access-key-id=STS.NT2Hdvsip2usKw7pJvt596Rh1&x-oss-expires=1737274199&x-oss-signature=2t2FAhdY0RCDobUyyGWWnj1Zn%2B2R5Fv5WNkkpWk9N%2FU%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>678CA57C86F9953738F9DBA7</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"10A77BCC0A84C72D17DF3A8F92539DCF"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2025-01-19 07:10:54 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/dSAoX6sq%2F889478%2F6735d73332841f81370642618cda50a015a68fc1%2F678ca547e9ac85879d704c479d576a3f2b452058?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5eHA97CnrZRhbeYSVGGlEojeLoV2ZfD0zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUjpMV73q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQzFAznsr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSigdhiM%2FXtuMkagAFz03W2W0QoSojL%2FQpfM%2Fmt0DiRa%2FfqDW%2FDkhwWuB5HzES7CM0Jt027qDwzeHMNmYwIwsQhLlayrL4T54mTEuJ%2B4pu86ZxKXIn9qWSKp0rl9%2FSspz0qYOBNa6ADiRPoP5n6ppidXE%2FTG4Mk5woMlTtqUo03b5%2BsYKVg%2F3Fd7PJjOiAA&uploadId=763A009C1EFE402A85BD4EC33CA70ABA&x-oss-access-key-id=STS.NT2Hdvsip2usKw7pJvt596Rh1&x-oss-expires=1737274199&x-oss-signature=2t2FAhdY0RCDobUyyGWWnj1Zn%2B2R5Fv5WNkkpWk9N%2FU%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>678CA57E78449133377FF80C</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"10A77BCC0A84C72D17DF3A8F92539DCF"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2025-01-19 07:10:55 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/dSAoX6sq%2F889478%2F6735d73332841f81370642618cda50a015a68fc1%2F678ca547e9ac85879d704c479d576a3f2b452058?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5eHA97CnrZRhbeYSVGGlEojeLoV2ZfD0zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUjpMV73q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQzFAznsr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSigdhiM%2FXtuMkagAFz03W2W0QoSojL%2FQpfM%2Fmt0DiRa%2FfqDW%2FDkhwWuB5HzES7CM0Jt027qDwzeHMNmYwIwsQhLlayrL4T54mTEuJ%2B4pu86ZxKXIn9qWSKp0rl9%2FSspz0qYOBNa6ADiRPoP5n6ppidXE%2FTG4Mk5woMlTtqUo03b5%2BsYKVg%2F3Fd7PJjOiAA&uploadId=763A009C1EFE402A85BD4EC33CA70ABA&x-oss-access-key-id=STS.NT2Hdvsip2usKw7pJvt596Rh1&x-oss-expires=1737274199&x-oss-signature=2t2FAhdY0RCDobUyyGWWnj1Zn%2B2R5Fv5WNkkpWk9N%2FU%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>678CA57ED867063534CBFCF0</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"10A77BCC0A84C72D17DF3A8F92539DCF"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2025-01-19 07:10:55 - ERROR - aliyundrive.py - 1077 - File tmp/124264779/124264779_p0.jpg Part 1 upload failed.
2025-01-19 07:11:51 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f017372707115882784e3c72","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:11:51 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:11:52 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 07:11:53 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f017372707128442908e3c72","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:11:53 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:11:55 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 07:11:55 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f017372707150973124e3c72","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:11:55 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:11:58 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 07:11:58 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f017372707183493475e3c72","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:11:58 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:12:02 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 07:12:02 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 07:12:02 - WARNING - aliyundrive.py - 831 - Token is about to expire, try to auto refresh.
2025-01-19 07:13:07 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Read timed out. (read timeout=30)")': /GmCvEeAj%2F889478%2F66a8ca2854610c49f6f44f2c9eea1332841a7098%2F678ca5e517d16812ff9748d0a1437fc1fea0cdbe?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5eHA97CnrZRhbeYSVGGlEojeLoV2ZfD0zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUjpMV73q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQzFAznsr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSigdhiM%2FXtuMkagAFz03W2W0QoSojL%2FQpfM%2Fmt0DiRa%2FfqDW%2FDkhwWuB5HzES7CM0Jt027qDwzeHMNmYwIwsQhLlayrL4T54mTEuJ%2B4pu86ZxKXIn9qWSKp0rl9%2FSspz0qYOBNa6ADiRPoP5n6ppidXE%2FTG4Mk5woMlTtqUo03b5%2BsYKVg%2F3Fd7PJjOiAA&uploadId=74CC2EAB015348B6A7E02D00EBCCA223&x-oss-access-key-id=STS.NT2Hdvsip2usKw7pJvt596Rh1&x-oss-expires=1737274357&x-oss-signature=GZw2Zzrzakst%2BjJ7vuOjpx6N6MUkTCpR0ph8PoO97ec%3D&x-oss-signature-version=OSS2
2025-01-19 07:18:12 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /HBQKCAF0%2F889478%2F675386058a465a331f184b269ec565dd49d93aaa%2F678ca716cbcba7579278476e8921bd14c9d214fe?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5eHA97CnrZRhbeYSVGGlEojeLoV2ZfD0zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUjpMV73q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQzFAznsr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSigdhiM%2FXtuMkagAFz03W2W0QoSojL%2FQpfM%2Fmt0DiRa%2FfqDW%2FDkhwWuB5HzES7CM0Jt027qDwzeHMNmYwIwsQhLlayrL4T54mTEuJ%2B4pu86ZxKXIn9qWSKp0rl9%2FSspz0qYOBNa6ADiRPoP5n6ppidXE%2FTG4Mk5woMlTtqUo03b5%2BsYKVg%2F3Fd7PJjOiAA&uploadId=4120B39D1C854D07BB177E2141509E74&x-oss-access-key-id=STS.NT2Hdvsip2usKw7pJvt596Rh1&x-oss-expires=1737274662&x-oss-signature=tH1CffM6NUe0jE%2BntLnhBl%2BxMRwsoeXwY87r56HWGJE%3D&x-oss-signature-version=OSS2
2025-01-19 07:23:25 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060e917372714054683675e0a6c","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:23:25 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:23:26 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 07:23:26 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060e917372714067403781e0a6c","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:23:26 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:23:28 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 07:23:29 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060e917372714090123972e0a6c","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:23:29 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:23:32 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 07:23:32 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060e917372714122854258e0a6c","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:23:32 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:23:36 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 07:23:36 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 07:25:05 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Read timed out. (read timeout=30)")': /ldiD9lze%2F889478%2F675933899c72e027c2b24395ac540bef4d238384%2F678ca8b32f69800acc8245ea960c4140b5585ab2?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fPHouBg5oV55aodHfJ1kIjfepmi%2FbmuDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVj9IFX3q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnewzFG7AMr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiTkHTcfXtuMkagAE6kvC7WvwPXTmJ4k%2BLmkUlLQdU9nWglXQWRH3jeEOHELJWinVzu82f%2B35nWrs5SqV5pToSO5taW6kZh9P%2FJHd5DtYGyVMbNeNZQl8siiR0si2ZFedEPxmFNj8mByEghrYEzM%2F57qnc%2BCwtRL%2B8VsCzEF9KkahYmO9nWi1LywkuKCAA&uploadId=D55549027496417DA8DB6AD42BA19195&x-oss-access-key-id=STS.NTzU15nE4PTCvQx2BvqeJd3MZ&x-oss-expires=1737275075&x-oss-signature=UmkB34f61dsE0avaILKizVqz%2FLCpswPcC2A8Pwf4V4M%3D&x-oss-signature-version=OSS2
2025-01-19 07:25:08 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/ldiD9lze%2F889478%2F675933899c72e027c2b24395ac540bef4d238384%2F678ca8b32f69800acc8245ea960c4140b5585ab2?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fPHouBg5oV55aodHfJ1kIjfepmi%2FbmuDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVj9IFX3q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnewzFG7AMr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiTkHTcfXtuMkagAE6kvC7WvwPXTmJ4k%2BLmkUlLQdU9nWglXQWRH3jeEOHELJWinVzu82f%2B35nWrs5SqV5pToSO5taW6kZh9P%2FJHd5DtYGyVMbNeNZQl8siiR0si2ZFedEPxmFNj8mByEghrYEzM%2F57qnc%2BCwtRL%2B8VsCzEF9KkahYmO9nWi1LywkuKCAA&uploadId=D55549027496417DA8DB6AD42BA19195&x-oss-access-key-id=STS.NTzU15nE4PTCvQx2BvqeJd3MZ&x-oss-expires=1737275075&x-oss-signature=UmkB34f61dsE0avaILKizVqz%2FLCpswPcC2A8Pwf4V4M%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>678CA8D289947D3136493B9A</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"6F3B8167F26563774F7AF3BD119FD3F4"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2025-01-19 07:25:08 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/ldiD9lze%2F889478%2F675933899c72e027c2b24395ac540bef4d238384%2F678ca8b32f69800acc8245ea960c4140b5585ab2?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fPHouBg5oV55aodHfJ1kIjfepmi%2FbmuDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVj9IFX3q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnewzFG7AMr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiTkHTcfXtuMkagAE6kvC7WvwPXTmJ4k%2BLmkUlLQdU9nWglXQWRH3jeEOHELJWinVzu82f%2B35nWrs5SqV5pToSO5taW6kZh9P%2FJHd5DtYGyVMbNeNZQl8siiR0si2ZFedEPxmFNj8mByEghrYEzM%2F57qnc%2BCwtRL%2B8VsCzEF9KkahYmO9nWi1LywkuKCAA&uploadId=D55549027496417DA8DB6AD42BA19195&x-oss-access-key-id=STS.NTzU15nE4PTCvQx2BvqeJd3MZ&x-oss-expires=1737275075&x-oss-signature=UmkB34f61dsE0avaILKizVqz%2FLCpswPcC2A8Pwf4V4M%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>678CA8D4CBC22F38324AA9A6</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"6F3B8167F26563774F7AF3BD119FD3F4"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2025-01-19 07:25:09 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/ldiD9lze%2F889478%2F675933899c72e027c2b24395ac540bef4d238384%2F678ca8b32f69800acc8245ea960c4140b5585ab2?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fPHouBg5oV55aodHfJ1kIjfepmi%2FbmuDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVj9IFX3q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnewzFG7AMr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiTkHTcfXtuMkagAE6kvC7WvwPXTmJ4k%2BLmkUlLQdU9nWglXQWRH3jeEOHELJWinVzu82f%2B35nWrs5SqV5pToSO5taW6kZh9P%2FJHd5DtYGyVMbNeNZQl8siiR0si2ZFedEPxmFNj8mByEghrYEzM%2F57qnc%2BCwtRL%2B8VsCzEF9KkahYmO9nWi1LywkuKCAA&uploadId=D55549027496417DA8DB6AD42BA19195&x-oss-access-key-id=STS.NTzU15nE4PTCvQx2BvqeJd3MZ&x-oss-expires=1737275075&x-oss-signature=UmkB34f61dsE0avaILKizVqz%2FLCpswPcC2A8Pwf4V4M%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>678CA8D47A795E39373BA1F4</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"6F3B8167F26563774F7AF3BD119FD3F4"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2025-01-19 07:25:09 - ERROR - aliyundrive.py - 1077 - File tmp/124691127/124691127_p0.png Part 1 upload failed.
2025-01-19 07:27:28 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701917372716484263232e3add","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:27:28 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:27:29 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 07:27:29 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701917372716497023360e3add","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:27:29 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:27:31 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 07:27:32 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701917372716519753679e3add","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:27:32 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:27:35 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 07:27:35 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701917372716552704018e3add","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:27:35 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:27:39 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 07:27:39 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 07:33:36 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06817372720159997316ea904","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:33:36 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:33:37 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 07:33:37 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06817372720172737475ea904","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:33:37 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:33:39 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 07:33:39 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06817372720195487774ea904","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:33:39 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:33:42 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 07:33:42 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06817372720228218315ea904","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:33:42 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:33:46 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 07:33:46 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 07:37:14 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /1lS884nj%2F889478%2F678b56b881ddea62f19f4f4eac7b231fd45e9f14%2F678cab8c20ebafea431d430eaa8bd1adcd418180?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fPHouBg5oV55aodHfJ1kIjfepmi%2FbmuDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVj9IFX3q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnewzFG7AMr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiTkHTcfXtuMkagAE6kvC7WvwPXTmJ4k%2BLmkUlLQdU9nWglXQWRH3jeEOHELJWinVzu82f%2B35nWrs5SqV5pToSO5taW6kZh9P%2FJHd5DtYGyVMbNeNZQl8siiR0si2ZFedEPxmFNj8mByEghrYEzM%2F57qnc%2BCwtRL%2B8VsCzEF9KkahYmO9nWi1LywkuKCAA&uploadId=AD633A5855A04DDD9D51DDD042079057&x-oss-access-key-id=STS.NTzU15nE4PTCvQx2BvqeJd3MZ&x-oss-expires=1737275804&x-oss-signature=i3rBXuUpSxG0Pv2ISTriIgH0%2FKYa%2FkK7EHjzQs20qPM%3D&x-oss-signature-version=OSS2
2025-01-19 07:42:50 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700917372725707842470eab9a","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:42:50 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:42:51 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 07:42:52 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700917372725720372549eab9a","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:42:52 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:42:54 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 07:42:54 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700917372725742822746eab9a","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:42:54 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:42:57 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 07:42:57 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700917372725775342921eab9a","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:42:57 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:43:01 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 07:43:01 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 07:45:15 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f017372727148673670e3c78","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:45:15 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:45:16 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 07:45:16 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f017372727161413805e3c78","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:45:16 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:45:18 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 07:45:18 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f017372727184194062e3c78","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:45:18 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:45:21 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 07:45:21 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f017372727216914354e3c78","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:45:21 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:45:25 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 07:45:25 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 07:45:43 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f017372727436076886e3c78","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:45:43 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:45:44 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 07:45:45 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f017372727448877005e3c78","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:45:45 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:45:47 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 07:45:47 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f017372727471647204e3c78","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:45:47 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:45:50 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 07:45:50 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f017372727504357608e3c78","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:45:50 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:45:54 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 07:45:54 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 07:51:57 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701917372731170376380e3ae0","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:51:57 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:51:58 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 07:51:58 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701917372731183116523e3ae0","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:51:58 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:52:00 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 07:52:00 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701917372731205836756e3ae0","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:52:00 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:52:03 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 07:52:04 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701917372731238577066e3ae0","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 07:52:04 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 07:52:08 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 07:52:08 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 07:53:58 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /GcmgCi0n%2F889478%2F6777902483a0c0fa5cd942ae87063594b221385f%2F678caf78d0b2de5d1af34f099c17d9621a87dabe?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fPHouBg5oV55aodHfJ1kIjfepmi%2FbmuDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVj9IFX3q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnewzFG7AMr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiTkHTcfXtuMkagAE6kvC7WvwPXTmJ4k%2BLmkUlLQdU9nWglXQWRH3jeEOHELJWinVzu82f%2B35nWrs5SqV5pToSO5taW6kZh9P%2FJHd5DtYGyVMbNeNZQl8siiR0si2ZFedEPxmFNj8mByEghrYEzM%2F57qnc%2BCwtRL%2B8VsCzEF9KkahYmO9nWi1LywkuKCAA&uploadId=D9E0018B51CA40BE9330CE6EC6C64C9D&x-oss-access-key-id=STS.NTzU15nE4PTCvQx2BvqeJd3MZ&x-oss-expires=1737276808&x-oss-signature=JGpywEkCeBs5i%2BKSVmYWxhTAx%2BxzYuWYG0PLE4eeYwc%3D&x-oss-signature-version=OSS2
2025-01-19 08:03:51 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /IwZNM3pB%2F889478%2F6728b53ed8a6efbebb0d4a3dbba9fba6cb2647f0%2F678cb1c9646e50f03d40450fb83829df9b8edcc1?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fPHouBg5oV55aodHfJ1kIjfepmi%2FbmuDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVj9IFX3q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnewzFG7AMr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiTkHTcfXtuMkagAE6kvC7WvwPXTmJ4k%2BLmkUlLQdU9nWglXQWRH3jeEOHELJWinVzu82f%2B35nWrs5SqV5pToSO5taW6kZh9P%2FJHd5DtYGyVMbNeNZQl8siiR0si2ZFedEPxmFNj8mByEghrYEzM%2F57qnc%2BCwtRL%2B8VsCzEF9KkahYmO9nWi1LywkuKCAA&uploadId=26FE92BA35DC485FA68D5C9B6C812075&x-oss-access-key-id=STS.NTzU15nE4PTCvQx2BvqeJd3MZ&x-oss-expires=1737277401&x-oss-signature=FnsmKq7GFR7GUmaOzH2tGphZP0VF3Vb%2BueGyMnnhbao%3D&x-oss-signature-version=OSS2
2025-01-19 08:05:48 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f617372739486396566e5d59","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 08:05:48 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 08:05:49 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 08:05:50 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f617372739499076676e5d59","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 08:05:50 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 08:05:52 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 08:05:52 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f617372739521836887e5d59","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 08:05:52 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 08:05:55 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 08:05:55 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f617372739554637244e5d59","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 08:05:55 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 08:05:59 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 08:05:59 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 08:06:58 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /DDyGatkB%2F889478%2F671388eda1c77d25bcfd492d9312798525740025%2F678cb28346b365a83be74c689c1e60a2f7272bd4?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fPHouBg5oV55aodHfJ1kIjfepmi%2FbmuDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVj9IFX3q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnewzFG7AMr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiTkHTcfXtuMkagAE6kvC7WvwPXTmJ4k%2BLmkUlLQdU9nWglXQWRH3jeEOHELJWinVzu82f%2B35nWrs5SqV5pToSO5taW6kZh9P%2FJHd5DtYGyVMbNeNZQl8siiR0si2ZFedEPxmFNj8mByEghrYEzM%2F57qnc%2BCwtRL%2B8VsCzEF9KkahYmO9nWi1LywkuKCAA&uploadId=B60A413C46A14B9DBBD160776BC4C16F&x-oss-access-key-id=STS.NTzU15nE4PTCvQx2BvqeJd3MZ&x-oss-expires=1737277587&x-oss-signature=ro0iYAv4rU6jNLRV2zzcpziyAJpEd1AJHzR6zNRiCDU%3D&x-oss-signature-version=OSS2
2025-01-19 08:08:25 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/10/17/07/30/02/123403199_p0.jpg
2025-01-19 08:13:46 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Read timed out. (read timeout=30)")': /6afbnipK%2F889478%2F675383180de30657a4794c3380629764962ce028%2F678cb41c344467cc6e8c48ae8f18acedc3a2b806?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fPHouBg5oV55aodHfJ1kIjfepmi%2FbmuDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVj9IFX3q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnewzFG7AMr7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiTkHTcfXtuMkagAE6kvC7WvwPXTmJ4k%2BLmkUlLQdU9nWglXQWRH3jeEOHELJWinVzu82f%2B35nWrs5SqV5pToSO5taW6kZh9P%2FJHd5DtYGyVMbNeNZQl8siiR0si2ZFedEPxmFNj8mByEghrYEzM%2F57qnc%2BCwtRL%2B8VsCzEF9KkahYmO9nWi1LywkuKCAA&uploadId=E860831057E042429614EDBB6646D414&x-oss-access-key-id=STS.NTzU15nE4PTCvQx2BvqeJd3MZ&x-oss-expires=1737277996&x-oss-signature=EzRBzf67uCp8sYBBK95UDP1c3DTWAOu24col8OXXfsw%3D&x-oss-signature-version=OSS2
2025-01-19 08:22:48 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/12/07/19/40/35/124979064_p0.jpg
2025-01-19 08:26:41 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f717372752011475787e69d4","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 08:26:41 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 08:26:42 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 08:26:42 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f717372752024045937e69d4","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 08:26:42 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 08:26:44 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 08:26:44 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f717372752046776157e69d4","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 08:26:44 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 08:26:47 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 08:26:48 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f717372752079346468e69d4","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 08:26:48 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 08:26:52 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 08:26:52 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 08:28:07 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2023/12/18/19/12/39/114336873_p0.jpg
2025-01-19 22:13:49 - WARNING - aliyundrive.py - 831 - Token is about to expire, try to auto refresh.
2025-01-19 22:15:13 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701917373249131951078e3ade","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:15:13 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:15:14 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 22:15:14 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701917373249144811130e3ade","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:15:14 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:15:16 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 22:15:16 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701917373249167651224e3ade","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:15:16 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:15:19 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 22:15:20 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701917373249200491352e3ade","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:15:20 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:15:24 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 22:15:24 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 22:28:40 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2023/11/12/12/41/53/113344698_p0.jpg
2025-01-19 22:33:35 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373260150728859e211f","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:33:35 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:33:36 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 22:33:36 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373260163618961e211f","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:33:36 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:33:38 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 22:33:38 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373260186431142e211f","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:33:38 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:33:41 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 22:33:42 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373260219301309e211f","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:33:42 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:33:46 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 22:33:46 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 22:38:31 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373263108665712e2126","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:38:31 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:38:32 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 22:38:32 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373263121435773e2126","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:38:32 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:38:34 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 22:38:34 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373263144305852e2126","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:38:34 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:38:37 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 22:38:37 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373263177056001e2126","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:38:37 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:38:41 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 22:38:41 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 22:39:42 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373263826798926e2126","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:39:42 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:39:43 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 22:39:44 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373263839618981e2126","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:39:44 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:39:46 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 22:39:46 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373263862391071e2126","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:39:46 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:39:49 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 22:39:49 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373263895351282e2126","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:39:49 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:39:53 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 22:39:53 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 22:45:56 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/04/05/14/16/15/117565680_p0.jpg
2025-01-19 22:49:16 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c817373269559846728e0969","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:49:16 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:49:17 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 22:49:17 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c817373269572706779e0969","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:49:17 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:49:19 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 22:49:19 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c817373269595476859e0969","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:49:19 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:49:22 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 22:49:23 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c817373269628456981e0969","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:49:23 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:49:27 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 22:49:27 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 22:52:01 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801d17373271209075307e69e0","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:52:01 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:52:02 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 22:52:02 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801d17373271221885345e69e0","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:52:02 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:52:04 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 22:52:04 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801d17373271244755421e69e0","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:52:04 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:52:07 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 22:52:07 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801d17373271277825556e69e0","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 22:52:07 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 22:52:11 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 22:52:11 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 23:00:12 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700917373276123945847eab98","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:00:12 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:00:13 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 23:00:13 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700917373276136705891eab98","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:00:13 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:00:15 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 23:00:16 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700917373276159556030eab98","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:00:16 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:00:19 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 23:00:19 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700917373276192406271eab98","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:00:19 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:00:23 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 23:00:23 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 23:00:29 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700917373276290786738eab98","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:00:29 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:00:30 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 23:00:30 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700917373276303556888eab98","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:00:30 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:00:32 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 23:00:32 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700917373276326337027eab98","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:00:32 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:00:35 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 23:00:36 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700917373276359117155eab98","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:00:36 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:00:40 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 23:00:40 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 23:08:16 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06717373280967472345ec749","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:08:16 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:08:17 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 23:08:18 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06717373280980332415ec749","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:08:18 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:08:20 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 23:08:20 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06717373281003092518ec749","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:08:20 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:08:23 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 23:08:23 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06717373281035952687ec749","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:08:23 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:08:27 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 23:08:27 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 23:21:17 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801517373288776051822eeda1","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:21:17 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:21:18 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 23:21:19 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801517373288788971890eeda1","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:21:19 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:21:21 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 23:21:21 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801517373288811811982eeda1","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:21:21 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:21:24 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 23:21:24 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801517373288844652140eeda1","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:21:24 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:21:28 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 23:21:28 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 23:23:07 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701917373289872797993e3adf","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:23:07 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:23:08 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 23:23:08 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701917373289886548084e3adf","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:23:08 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:23:10 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 23:23:11 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701917373289910388186e3adf","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:23:11 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:23:14 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 23:23:14 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701917373289944028367e3adf","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:23:14 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:23:18 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 23:23:18 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 23:28:42 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701417373293218686003e5e3c","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:28:42 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:28:43 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 23:28:43 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701417373293231576098e5e3c","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:28:43 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:28:45 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 23:28:45 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701417373293254466210e5e3c","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:28:45 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:28:48 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 23:28:48 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701417373293287296378e5e3c","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:28:48 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:28:52 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 23:28:52 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 23:30:23 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801c17373294233105545e639e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:30:23 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:30:24 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 23:30:24 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801c17373294245875622e639e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:30:24 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:30:26 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 23:30:27 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801c17373294268655745e639e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:30:27 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:30:30 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 23:30:30 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801c17373294301625895e639e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:30:30 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:30:34 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 23:30:34 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 23:37:34 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373298548221393e2126","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:37:34 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:37:35 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 23:37:36 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373298561021478e2126","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:37:36 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:37:38 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 23:37:38 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373298583801673e2126","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:37:38 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:37:41 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 23:37:41 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373298616611853e2126","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:37:41 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:37:45 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 23:37:45 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 23:38:26 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373299064284504e2126","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:38:26 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:38:27 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 23:38:27 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373299077074585e2126","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:38:27 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:38:29 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 23:38:30 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373299100074737e2126","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:38:30 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:38:33 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 23:38:33 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373299132824973e2126","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:38:33 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:38:37 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 23:38:37 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 23:41:03 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f617373300636615286e5d5c","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:41:03 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:41:04 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 23:41:05 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f617373300649485353e5d5c","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:41:05 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:41:07 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 23:41:07 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f617373300672325452e5d5c","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:41:07 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:41:10 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 23:41:10 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f617373300705175633e5d5c","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:41:10 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:41:14 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 23:41:14 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 23:49:06 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/05/21/18/29/35/118925865_p0.jpg
2025-01-19 23:50:17 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060e917373306170445408e0a68","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:50:17 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:50:18 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-19 23:50:18 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060e917373306183235492e0a68","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:50:18 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:50:20 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-19 23:50:21 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701917373306211138620e3ae0","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:50:21 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:50:24 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-19 23:50:24 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701917373306243988854e3ae0","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-19 23:50:24 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-19 23:50:28 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-19 23:50:28 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-19 23:53:06 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/08/24/19/11/41/121792450_p0.jpg
2025-01-20 00:04:50 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801717373314904091032eb4e3","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:04:50 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:04:51 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-20 00:04:51 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801717373314916921177eb4e3","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:04:51 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:04:53 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-20 00:04:54 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060ea17373314947598351ef470","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:04:54 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:04:57 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-20 00:04:58 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060ea17373314980608492ef470","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:04:58 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:05:02 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-20 00:05:02 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-20 00:07:06 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/11/19/00/00/45/124430037_p0.jpg
2025-01-20 00:10:47 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/11/11/15/43/50/124196118_p0.png
2025-01-20 00:12:49 - WARNING - aliyundrive.py - 831 - Token is about to expire, try to auto refresh.
2025-01-20 00:12:56 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f717373319762463428e69ca","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:12:56 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:12:57 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-20 00:12:57 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f717373319776363540e69ca","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:12:57 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:12:59 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-20 00:13:00 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f717373319800623770e69ca","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:13:00 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:13:03 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-20 00:13:03 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060f717373319834464092e69ca","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:13:03 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:13:07 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-20 00:13:07 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-20 00:18:02 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801717373322826488163eb4dc","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:18:02 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:18:03 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-20 00:18:04 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801717373322840878294eb4dc","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:18:04 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:18:06 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-20 00:18:07 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801717373322866308563eb4dc","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:18:07 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:18:10 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-20 00:18:10 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801717373322901438926eb4dc","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:18:10 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:18:14 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-20 00:18:14 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-20 00:19:29 - ERROR - base.py - 94 - https://api.aliyundrive.com/v2/file/complete:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2025-01-20 00:19:30 - WARNING - base.py - 25 - Retry func _post_file_complete 1 time.
2025-01-20 00:23:11 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801617373325916936875ee099","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:23:11 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:23:12 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-20 00:23:13 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801617373325929826980ee099","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:23:13 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:23:15 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-20 00:23:15 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801617373325952677159ee099","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:23:15 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:23:18 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-20 00:23:18 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801617373325985727435ee099","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:23:18 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:23:22 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-20 00:23:22 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-20 00:28:43 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060e817373329229187406e4106","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:28:43 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:28:44 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-20 00:28:44 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0060e817373329242037531e4106","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:28:44 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:28:46 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-20 00:28:47 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701017373329269948943e8e5b","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:28:47 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:28:50 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-20 00:28:50 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701017373329302771171e8e5b","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:28:50 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:28:54 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-20 00:28:54 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-20 00:32:34 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c917373331542503174ed787","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:32:34 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:32:35 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-20 00:32:35 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c917373331555413351ed787","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:32:35 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:32:37 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-20 00:32:38 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c917373331578253648ed787","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:32:38 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:32:41 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-20 00:32:41 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a0c917373331611363983ed787","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:32:41 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:32:45 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-20 00:32:45 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-20 00:38:18 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801317373334980202906e472b","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:38:18 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:38:19 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-20 00:38:19 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801317373334993083043e472b","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:38:19 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:38:21 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-20 00:38:21 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801317373335016113233e472b","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:38:21 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:38:24 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-20 00:38:25 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801317373335048913538e472b","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:38:25 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:38:29 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-20 00:38:29 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
2025-01-20 00:46:51 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373340117853812e212d","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:46:51 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:46:52 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-20 00:46:53 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373340130643936e212d","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:46:53 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:46:55 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-20 00:46:55 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373340153464176e212d","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:46:55 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:46:58 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-20 00:46:58 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00a06617373340187024505e212d","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-20 00:46:58 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-20 00:47:02 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-20 00:47:02 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/7847900/しゅにち|1日目東A-47ab - 7847900.
